src/dialogs/NameRequestErrorDialog.vue

```diff
@@ -80,8 +76,17 @@ export default class NameRequestErrorDialog extends Vue {
   /** Enum definition for use in template. */
   readonly NameRequestStates = NameRequestStates
 
+  /** True if a match was not found for the NR number and provided phone/email. */
+  get isMatchNotFound (): boolean {
+    return (
+      this.type === NameRequestStates.NOT_FOUND ||
+      this.type === ('INCORRECT_CONTACT' as NameRequestStates) ||
+      this.type === ('NO_CONTACT' as NameRequestStates)
```

Comment:
I deleted the local NameRequestStates enum so I could use the shared version, which doesn't contain these values.

Don't worry, soon we won't need these values anyway.